Documentation/articles/packaging_games.md

```diff
@@ -1,10 +1,14 @@
+# Packaging games built with MonoGame
+
 Once your game is ready to be published, it is recommended that you package it properly for consumption by players.
 
 ## Desktop games
 
 To publish desktop games, it is recommended that you build your project as a [self-contained](https://docs.microsoft.com/en-us/dotnet/core/deploying/#publish-self-contained) .NET Core app. As such, your game will require absolutely no external dependencies and should run out-of-the-box as-is.
 
-### Building and packaging for Windows
+---
+
+## Building and packaging for Windows
```

Comment:
I think these were supposed to go under the Desktop games section.

Reply:
Updated